onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -3456,10 +3456,37 @@ def aten_meshgrid(tensors: Sequence[TensorType]) -> TensorType:
     raise NotImplementedError()
 
 
-def aten_min(self: TensorType) -> TensorType:
+@torch_op("aten::min")
+def aten_min(self: TReal) -> TReal:
     """min(Tensor self) -> Tensor"""
 
-    raise NotImplementedError()
+    return op.ReduceMin(self, keepdims=0)
+
+
+@torch_op("aten::min", overload=True)
+def aten_min_dim(self: TReal, dim: int, keepdim: bool = False) -> Tuple[TReal, TInt]:
+
+    neg_1 = op.Constant(value_ints=[-1])
```

Comment:
If input is rank 0, can we directly return self and index 0? Because there is only one element anyways? Better would there be a way to remove if? I see it’s a little tricky because we need to return index as well. Still thinking. Other parts LGTM

Reply:
```python
    self_rank = op.Size(op.Shape(self))
    if self_rank == 0:
        result = self
    else:
        ...
```

Reply:
This if condition is known compile time so I think its fine